### PR TITLE
Use `mvn hpi:run -Djetty.contextPath=/bob` to change context path

### PR DESCRIPTION
**Pull request by i386**
# Description

`cd blueocean && mvn hpi:run -Djetty.contextPath=/bob`

then browse to `http://localhost:8080/bob`

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean/pom.xml [hunk 1]
```diff
@@ -15,6 +15,10 @@
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Blue+Ocean+Plugin</url>
     <description>Blue Ocean is a new project that rethinks the user experience of Jenkins. Designed from the ground up for Jenkins Pipeline and compatible with Freestyle jobs, Blue Ocean reduces clutter and increases clarity for every member of your team.</description>
 
+    <properties>
+        <jetty.contextPath>/${project.artifactId}</jetty.contextPath>
```
Comment (kzantow) [inline blueocean/pom.xml]: this should probably be `/jenkins` right?

Comment (kzantow) [inline blueocean/pom.xml]: ... and does `-Djetty.contextPath=/` work for the root URL?

**Comment by i386** [comment]
@kzantow Fixed it so it is now `/jenkins`

Yes, everything seems to work just fine when `/` 👍

**Comment by i386** [comment]
@michaelneale FYI, this now works

**Comment by tfennelly** [comment]
Yes, sometimes it's easier to go around if you can't get through !!

**Pull request merged**